jax/_src/numpy/lax_numpy.py

```diff
@@ -972,9 +972,27 @@ def flipud(m: ArrayLike) -> Array:
   util.check_arraylike("flipud", m)
   return _flip(asarray(m), 0)
 
-@util.implements(np.iscomplex)
 @jit
 def iscomplex(x: ArrayLike) -> Array:
+  """Return a new array containing True where the input elements are complex 
```

Comment:
One-line summary should fit on one line; maybe "Return boolean array showing where the input is complex"